_assets/stylesheets/vendor/_normalize.scss

```diff
@@ -377,9 +378,9 @@ input[type="search"]::-webkit-search-decoration {
  */
 
 fieldset {
-  border: 1px solid #c0c0c0;
+  padding: .35em .625em .75em;
   margin: 0 2px;
-  padding: 0.35em 0.625em 0.75em;
+  border: 1px solid #c0c0c0;
```

Comment:
Color literals like `#c0c0c0` should only be used in variable declarations; they should be referred to via variable everywhere else.